tools/functional-tester/etcd-runner/main.go

```diff
@@ -227,7 +227,7 @@ func runWatcher(eps []string) {
 	keyPrePrefix := 30              // max number of keyPrePrefixs for put operation
 
 	prefixes := generateUniqueKeys(5, noOfPrefixes)
-	keys := generateRandomKeys(10, keyPrePrefix)
+	keys := generateUniqueKeys(10, keyPrePrefix)
```

Comment:
I mean why do we need to change this to unique key?

As you just said

> since we are doing revision check for not performing put twice for the same keys. So even now if random keys have duplicate keys it will work.

I assume you agree random keys should work now? So we do not need to change this line?